Classes_Homework5/README.md

```diff
@@ -0,0 +1,6 @@
+1. PyInputPlus module needs to be installed to run this app: https://pypi.org/project/PyInputPlus/
```

Comment:
The python way to manage dependencies and libs is using the **requirements.txt** file.
More about that:
https://note.nkmk.me/en/python-pip-install-requirements/